apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -546,6 +547,8 @@ class TopInstructions extends Component {
             instructions={longInstructions}
             onResize={this.adjustMaxNeededHeight}
             inTopPane
+            isImmersiveButtonHasRoundBorders
```

Comment:
We need that prop where we don't have `InlineAudio` next to `ImmersiveReaderButton`. And it's turned out that that's the cases in pr when it happens

Reply:
![image](https://user-images.githubusercontent.com/22244040/228318202-5fd7857f-6afd-46b6-965f-52271cbff820.png)
Here's the perfect ilustration of when we need rounded borders.
Half of the button has no border radius when they are not standing next of each other.
The root cause of this is that since the difference with redesign is big, it's impossible to use the same approach for new styles

Reply:
Also note that `isImmersiveButtonHasRoundBorders` = (equals) `isImmersiveButtonHasRoundBorders={true}`
and we don't pass `isImmersiveButtonHasRoundBorders` to `<Instructions/>` it'll be the same as `isImmersiveButtonHasRoundBorders={false}`